.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 rvm:
-  - "1.9.3"
```

Comment:
We are still running RHEL6 derivative on pleny of our systems with ruby 1.9.
Deprecation of ruby 1.9 will be painful to me.